### PR TITLE
Add more metadata options to notifications

### PR DESCRIPTION
**Pull request by JonnyWong16**
Reference #273

### CODE REVIEW

File: plexpy/notifiers.py [hunk 1]
```diff
@@ -1418,10 +1418,10 @@ def return_config_options(self):
         config_option = [{'label': 'Ifttt Maker Channel Key',
                           'value': self.apikey,
                           'name': 'ifttt_key',
-                          'description': 'Your Ifttt  key. You can get a key from here https://ifttt.com/maker',
+                          'description': 'Your Ifttt  key. You can get a key from <a href="https://ifttt.com/maker">here</a>.',
```
Comment (drzoidberg33) [inline plexpy/notifiers.py]: May want to set `target="_blank"` here.

File: plexpy/notifiers.py [hunk 2]
```diff
@@ -1490,7 +1490,7 @@ def return_config_options(self):
         config_option = [{'label': 'Telegram Bot Token',
                           'value': self.bot_token,
                           'name': 'telegram_bot_token',
-                          'description': 'Your bot token. Contact <a href="http://telegram.me/BotFather">@BotFather</a> on Telegram to get one.',
+                          'description': 'Your Telegram bot token. Contact <a href="http://telegram.me/BotFather">@BotFather</a> on Telegram to get one.',
```
Comment (drzoidberg33) [inline plexpy/notifiers.py]: Same here.

**Comment by JonnyWong16** [comment]
Should be good.

**Comment by drzoidberg33** [comment]
Thanks!

**Pull request merged**